scrum/policies.md

```diff
@@ -226,6 +227,9 @@ These metrics are then recorded into the "Sprint Summary Card" at the beginning
 Card
 : This refers to a Trello Card. An example card had been created here to understand its parts.
 
+Custom Field
+: When opening a card, there are modules that can be activated for that card. One of them allows for a new field that can contain any information the card owner requires. Usually, this is used to keep track of a card's 'total' Story Points.
```

Comment:
This is a little generic, is it worth adding an image to help?

Reply:
I agree now that you say that. I will add one in. Thank you!